src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+import duke.exception.EmptyDescription;
+import duke.exception.InvalidTypeOfTask;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Duke program maintains a taskList for user to track tasks.
+ * Reads user input tasks(todo, event, deadline).
+ * Able to perform add, delete, markasDone tasks.
+ *
+ * @author Oh Jun Ming
+ * @version 1.0
+ */
+
+public class Duke {
+    private Storage storage;
+    private Ui ui;
+    private TaskList taskList;
+
+    Duke() {
+        ui = new Ui();
+        storage = new Storage();
+        try {
+            taskList = storage.load();
+        } catch (IOException e) {
+            System.out.println("file not found");
+        }
+    }
+
+    /**
+     * Initialise scanner.
+     *
+     * @param args
+     */
+    public static void main(String[] args){
+        Duke duke = new Duke();
+        duke.execute();
+    }
+
+    /**
+     * Start Duke chat services.
+     */
+    public void execute() {
+        ui.Greet();
+        Boolean shouldExit = false;
+        Scanner s = new Scanner(System.in);
+
+        while(!shouldExit && s.hasNextLine()){
```

Comment:
Whitespace before {. There are similar cases in other files as well, I will not point out all of them.